kafka-connect-aws-s3/src/fun/scala/io/lenses/streamreactor/connect/S3CompressionTest.scala

```diff
@@ -46,40 +52,38 @@ class S3CompressionTest
 
   behavior of "AWS S3 connector"
 
-  val compressionCodecTestCases = Table(
+  val compressionCodecTestCases: TableFor4[String, String, Boolean, Seq[String]] = Table(
     ("format", "codec", "configure", "extraCommands"),
-    ( "avro", "uncompressed", false, Seq.empty[String]),
-    ( "avro", "deflate", true, Seq.empty[String]),
-    ( "avro", "bzip2", false, Seq.empty[String]),
-    ( "avro", "snappy", false, Seq.empty[String]),
-    ( "avro", "xz", true, Seq.empty[String]),
-    ( "avro", "zstd", true, Seq.empty[String]),
+    ("avro", "uncompressed", false, Seq.empty[String]),
+    ("avro", "deflate", true, Seq.empty[String]),
+    ("avro", "bzip2", false, Seq.empty[String]),
+    ("avro", "snappy", false, Seq.empty[String]),
+    ("avro", "xz", true, Seq.empty[String]),
+    ("avro", "zstd", true, Seq.empty[String]),
     ("parquet", "uncompressed", false, Seq.empty[String]),
     ("parquet", "snappy", false, Seq.empty[String]),
     ("parquet", "gzip", false, Seq.empty[String]),
-   // ("parquet", "lzo", false, Seq("lzo")),
-   // brotli lib is madly out of date and I do not believe we can support it
-   // ("parquet", "brotli", false, Seq.empty[String]),
-   // ("parquet", "lz4", false, Seq.empty[String]),
+    // ("parquet", "lzo", false, Seq("lzo")),
```

Comment:
Is there a reason to comment out these? and if so, can we just remove - the git history has the info?

Reply:
technical debt, they should be investigated and fixed some day.

2 uncommented and 2 removed, along with some extra unused code.